PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-azure-sdk/src/main/java/com/microsoft/azure/toolkit/intellij/azure/sdk/buildtool/LoadJsonConfigFile.java

```diff
@@ -23,22 +30,31 @@ public class LoadJsonConfigFile {
     private static LoadJsonConfigFile instance;
     private JSONObject jsonObject;
```

Comment:
Can this be final?

Reply:
private final JSONObject jsonObject? 
jsonObject is now initialized when the instance is created -- had to make a temporary JSON Object in the loadJsonObject method that's returned to final JSONObject jsonObject